users/tests/test_views.py

```diff
@@ -496,3 +497,32 @@ def test_menu_expansion_update_should_update_menu_expansion_setting_in_user_pref
 
         user_menu_expansion_setting_after_update = CustomUserPreferences.objects.get(user=self.employee).expanded_menu
         self.assertEqual(user_menu_expansion_setting_after_update, not expanded_menu)
+
+
+class ResetPasswordTests(TestCase):
+    def setUp(self) -> None:
+        self.user_password = "Password1234!"
+        self.user = UserFactory()
+        self.user.set_password(self.user_password)
+        self.user.save()
+
+        reset_password_response = self.client.post(reverse("password_reset"), data={"email": self.user.email})
+        self.uid = reset_password_response.context["uid"]
+        token = reset_password_response.context["token"]
+        reset_password_confirm_response = self.client.post(
+            reverse("password_reset_confirm", kwargs={"uidb64": self.uid, "token": token})
+        )
+        self.url = reset_password_confirm_response.url
```

Comment:
It's not a best place for such things. You should create separate method in this test class and just call method with these API calls in this method. `setUp` method should be used only for variables initiation. We're losing some context when you adding such things in `setUp`. Also at least you should such logic move to separate method.

Reply:
Done